src/main/java/picocli/CommandLine.java

```diff
@@ -1104,6 +1104,31 @@ public CommandLine setUnmatchedOptionsArePositionalParams(boolean newValue) {
         return this;
     }
 
+    /** Returns whether positional parameters on the command line are allowed to occur before the special End of Options delimiter.
+     * The default is {@code true}.
+     * @return {@code true} positional parameters may occur anywhere on the command line, {@code false} if they must follow End of Options.
+     */
```

Comment:
Can you add `@since 4.8.0` to the javadoc for all public methods that were added in this pull request?

(Picocli follows semantic versioning, so additional API means the next version number should be 4.8.0, not 4.7.6.)